httpony/server.py

```diff
@@ -9,9 +9,16 @@
 
 
 def main():
+    """Read arguments if given"""
+    parser = argparse.ArgumentParser()
+    parser.add_argument("-l", "--listen", help="Set ip/hostname for HTTPony to use",
+                        default="localhost")
```

Comment:
Here's a cool style tip that I picked up along the way that I think really helps legibility. It has to do with what should be done when there are so many arguments that a line should wrap. PEP 8 calls it a "hanging indent" (https://www.python.org/dev/peps/pep-0008/#indentation). Consider the following example:
```python
my_boring_object.run_a_long_method_name(foo=True, bar=False, baz=None,
                                        message='Hi mom')
another.execute_bad_api(asdf='This argument does not matter', fizz=True,
                        buzz=False)
final_object.fire_some_obnoxious_event_now(wow='Please stop torturing me',
                                           when='now! do it now!')
```
versus:
```python
my_boring_object.run_a_long_method_name(
    foo=True, bar=False, baz=None, message='Hi mom')
another.execute_bad_api(
    asdf='This argument does not matter', fizz=True, buzz=False)
final_object.fire_some_obnoxious_event_now(
    wow='Please stop torturing me', when='now! do it now!')
```
The code is exactly the same, but I think that the second version is way easier to read. The reason is that your eye doesn't have to scan to find where the arguments start. If you always default to a hanging indent when arguments need to wrap lines, then you know exactly where your eye wants to put things.